zstacklib/zstacklib/utils/sharedBlockStorageDriver.py

```
@@ -0,0 +1,49 @@
+
+        with lvm.RecursiveOperateLv(metadata_path, shared=False):
+            # Create metadata header
+            header = BlkMetaHeader(magic=META_MAGIC, length=len(imf))
+
+            # Create metadata
+            meta = BlkMeta(header=header, data=imf)
+
+            # Write to the logical volume
+            self.write_meta(metadata_path, meta)
+
+    def read_metadata(self, metadata_path):
```

Ruff (0.14.14)

Unused method argument: `metadata_path`

(ARG002)